vms/traffic-gen/scripts/build-vm-image

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# This file is part of the kiagnose project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 Red Hat, Inc.
+#
+
+export LIBGUESTFS_BACKEND=direct
+
+virt-builder centosstream-8 \
+  --format qcow2 \
+  --root-password password:redhat \
+  --install cloud-init,driverctl,tuned-profiles-cpu-partitioning,tar,python3,pciutils \
```

Comment:
aren't we missing a lot of packages? https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/main/traffic-gen/Dockerfile#L7

Reply:
Is there a package you saw that is missing after manually running TRex?

Reply:
I didn't run it yet. In order to run it one would need to add the trex scripts